wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -126,6 +125,7 @@ def config
 
     inside "config" do
       template "application.rb"
+      template 'routes.erb', "routes.rb"
```

Comment:
I guess we can still do that with a routes.rb in the wagn gem.  This is a really small routes file and it gives them a good place to set the mount point (relative root).